src/editor/Neovim/Input.re

```diff
@@ -0,0 +1,50 @@
+/*
+ * Input.re
+ *
+ * Input conversion for Neovim
+ */
+
+open Revery_Core;
+
+type t = string;
+
+let keyToCharacter = (key: Key.t) => switch(key) {
+      | Key.KEY_TAB => Some("TAB")
+      | Key.KEY_BACKSPACE => Some("BS")
+      | Key.KEY_ENTER => Some("CR")
+      | Key.KEY_ESCAPE => Some("ESC")
+      | Key.KEY_RIGHT_SHIFT
+      | Key.KEY_LEFT_SHIFT => Some("SHIFT")
+      | Key.KEY_UP => Some("UP")
+      | Key.KEY_LEFT => Some("LEFT")
+      | Key.KEY_RIGHT => Some("RIGHT")
+      | Key.KEY_DOWN => Some("DOWN")
+      | _ => None
+};
+
+let keyPressToString = (~altKey, ~shiftKey, ~ctrlKey, ~superKey, s) => {
+    let s = s == "<" ? "lt" : s;
+    
+    let s = ctrlKey ? "c-" ++ s : s;
+    let s = shiftKey ? "s-" ++ s : s;
+    let s = altKey ? "a-" ++ s : s;
+    let s = superKey ? "m-" ++ s : s;
```

Comment:
Neovim doc says:
```
<S-...>		shift-key			*shift* *<S-*
<C-...>		control-key			*control* *ctrl* *<C-*
<M-...>		alt-key or meta-key		*META* *ALT* *<M-*
<A-...>		same as <M-...>			*<A-*
<D-...>		command-key or "super" key	*<D-*
```
So the last line should be `let s = superKey ? "d-" ++ s : s;`

Reply:
Good catch @romgrk - thank you!